### DISCUSSION TITLE
mygames.xml edit

### DISCUSSION THREAD

**Discussion by Manfred282** [aldostools/webMAN-MOD#583] [Q&A] [answered]
Hi there,

i have the problem, that the sorting order for games are messed up, because webMAN MOD only uses the first characters for sorting.
So my games are like "Rock Band 3, Rock Band, Rock Band 2" or "LittleBigPlanet 2, LittleBigPlanet, LittleBigPlanet 3".

I want to edit the mygames.xml, scan content at startup is already disables. But there is one big problem.

The mygames.xml is not formated, its simply 1 line if i open the file with Notepad++.
I did try several XML Formatters, but they didnt work, its still messed up.

So whats the best way to format the mygames.xml, because i dont know whats the starting and ending point is for the entries .

Some help would be nice, thx in advance ;)

**Comment by aldostools** [comment, marked as answer]
I made this quick tool 
[FormatXML.zip](https://github.com/aldostools/webMAN-MOD/files/7576209/FormatXML.zip)

It needs this runtime. Install it if you get an error about MSVBVM50.DLL
http://download.microsoft.com/download/vb50pro/utility/1/win98/en-us/msvbvm50.exe

Extract the zip, then drag & drop mygames.xml over the FormatXML.exe

It will output the file: mygames.xml.formatted.xml (you will have to rename it back to mygames.xml)

**Reply by Manfred282** [reply]
great, works like a charm ;)

thx again and keep the good work
